src/components/carousel/carousel.js

```diff
@@ -0,0 +1,63 @@
+import mixin from '../../globals/js/misc/mixin';
+import createComponent from '../../globals/js/mixins/create-component';
+import initComponentBySearch from '../../globals/js/mixins/init-component-by-search';
+// import eventMatches from '../../globals/js/misc/event-matches';
```

Comment:
Nitpick 🙇 - This line should be removed (by when this work completes).

Reply:
fixed! thank you